years_to_hours.rb

```diff
@@ -3,14 +3,14 @@
 years = years.to_i # this converts a string to an integer
 hours = years * 365 * 24
 puts "That's #{hours} hours."
-#converting minutes to minutes
-puts "Enter a number of minutes"
+#converting decades to minutes
+puts "Enter a number of decades"
 minutes = gets.chomp
 minutes minutes.to_i
```

Comment:
I got an error on this line -- I'm guessing it's a typo, it seems like you understand the concepts.